suite-native/forms/src/Form.tsx

```diff
@@ -22,7 +22,14 @@ export const Form = <TFieldValues extends FieldValues>({
     // TODO: once react-hook-form is upgraded to v7 remove errors, because it will be accessible via useControl.
     // It will same some unnecessary rerenders. Also FormContext from rhf should't be used because
     // there is lof of unnecessary stuff that will cause extra rerenders, but we need only control.
-    const formContextValue = { control: form.control, errors: form.errors };
+    const formContextValue = {
+        control: form.control,
+        errors: form.formState.errors,
+    };
```

Comment:
Please delete this comment and remove `errors` field from context value. In `suite-native/forms/src/hooks/useField.ts` you are now extracting errors from fieldState, so it shouldn't be necessary here.

Reply:
Done: 58c78827ee4f5661b55510d92452bd051f24e3e9